src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -113,10 +117,11 @@ public ProAI(final String name, final String type) {
         new ProPurchaseAI(utils, battleUtils, transportUtils, attackOptionsUtils, territoryValueUtils, purchaseUtils);
     retreatAI = new ProRetreatAI(this, battleUtils);
     scrambleAI = new ProScrambleAI(this, battleUtils, attackOptionsUtils);
-    politicsAI = new ProPoliticsAI(this, utils, attackOptionsUtils);
+    politicsAI = new ProPoliticsAI(this, utils, battleUtils, attackOptionsUtils);
     data = null;
     storedCombatMoveMap = null;
     storedPurchaseTerritories = null;
+    storedStrafingTerritories = new ArrayList<Territory>();
```

Comment:
Yep. I hate inconsistency though. I'll probably look to make a pass over all the AI code to add in Java 7/8 features where appropriate.